frontend/src/common/utils/utils.ts

```diff
@@ -2,3 +2,5 @@
 export function classNames(...classes: any[]) {
   return classes.filter(Boolean).join(' ');
 }
+
+export const NAME_REGEX = /^(?!.*[ ]{2})[\w-\săîâșțĂÎÂȘȚ]*$/;
```

Comment:
we should do a different file in /constants called patterns.ts or regex.ts (up to you). In there we should have
export const REGEX = {
  NAME: ....
  PHONE: ....
}

Reply:
Renamed to patterns.ts